package.json

```diff
@@ -14,30 +14,30 @@
     "keywords": [
         "visual studio code",
         "extension",
-        "delve language",
+        "rel language",
         "syntax highlighter",
-        "delve syntax highlighter"
+        "rel syntax highlighter"
     ],
     "categories": [
         "Programming Languages"
     ],
     "activationEvents": [
-		"onLanguage:delve"
+		"onLanguage:rel"
 	],
     "main": "extension.js",
     "contributes": {
         "languages": [{
-            "id": "delve",
-            "aliases": ["Delve", "delve"],
-            "extensions": [".delve"],
+            "id": "rel",
+            "aliases": ["Rel", "rel"],
+            "extensions": [".rel", ".delve"],
             "configuration": "./language-configuration.json"
         }],
         "grammars": [{
-            "language": "delve",
-            "scopeName": "source.delve",
-            "path": "./syntaxes/delve.tmLanguage.json"
+            "language": "rel",
+            "scopeName": "source.rel",
+            "path": "./syntaxes/rel.tmLanguage.json"
         }]
     },
     "publisher": "RelationalAI",
-    "repository": "https://github.com/RelationalAI-oss/Delve_VSCode"
+    "repository": "https://github.com/RelationalAI-oss/Rel_VSCode"
```

Comment:
I guess this will break until we rename the repo. Probably better to rename the repo _first_, since i think GitHub will automatically redirect people using the old URL to the new one anyway.

We're filing a ticket with IT to rename this repo now.

(So nothing for you to change here, just please wait to merge until this is done.)

Reply:
https://github.com/RelationalAI/it-ops/issues/170

Reply:
Yeah, great, thanks! Also, I am not familiar with the process of publishing the VSCode extension in a central repository either, so I did not do anything in that direction.

Reply:
hokay, the repo has been renamed, so i think this should be good to go.


> I am not familiar with the process of publishing the VSCode extension in a central repository either

:/ i'm not sure either! :o 

It would be great if we can somehow test the deployment before merging this PR.... Unfortunately i don't think anyone knows how to run it 😬

Reply:
The `vsce package` command runs fine, and I was able to install the extension locally from the packaged .vsix file.

I have gone through [the instructions here about publishing an extension](https://code.visualstudio.com/api/working-with-extensions/publishing-extension#publishing-extensions). I imagine all the prerequisites are met since this extension, although with different name, has been published already, i.e., there already exists a publisher ID and the necessary credentials to run the `vsce publish 1.0.7` manually (at least, I don't see any GH workflow set up for the repository).

Reply:
Amazing, wow, thank you for looking into all of that!!

If you were able to install it and it worked okay, that sounds like good enough for me! :) I'll merge this and then we can see about how to publish it. Thanks @imbur!